Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/ComponentModeDelegate.h

```diff
@@ -77,6 +77,13 @@ namespace AzToolsFramework
             /// The function to call when this ComponentModeDelegate detects an event to enter ComponentMode.
             void SetAddComponentModeCallback(
                 const AZStd::function<void(const AZ::EntityComponentIdPair&)>& addComponentModeCallback);
+            
+            /// Is the EntityComponentIdPair connected to the ComponentModeDelegate.
+            bool IsConnected() const
+            {
+                return m_entityComponentIdPair != AZ::EntityComponentIdPair(AZ::EntityId(), AZ::InvalidComponentId) &&
+                    !(m_componentType.IsNull());  
```

Comment:
nit: We don't need the extra parens around this (so just `!m_componentType.IsNull()` is fine)